core/unix/inc/TUnixSystem.h

```diff
@@ -31,11 +31,30 @@
 #include "TTimer.h"
 #endif
 
+#include <thread>
+#include <memory>
+
+int StackTraceExec(void *);
+
 typedef void (*SigHandler_t)(ESignals);
 
+struct TStackTraceHelper {
+   static const int             stringLength = 255;
+   char                         shellExec[stringLength];
+   char                         pidString[stringLength];
+   char                         pidNum[stringLength];
+   int                          parentToChild[2];
+   int                          childToParent[2];
```

Comment:
int should be referred to as 'Int_t'
char should be referred to as 'Char_t' 
(thought out the code).
the exception is 'const char*' (correctly spelled as 'const char*')